crawler/utils.py

```diff
@@ -16,16 +16,7 @@ def send_message(self, sent_str):
         try:
             client.chat_postMessage(
                 channel=os.getenv("SLACK_CHANNEL_ID", ""),
-                blocks=[
-                    {
-                        "type": "section",
-                        "text": {
-                            "type": "mrkdwn",
-                            "text": "Danny Torrence left the following review for your property:",
-                        },
-                    },
-                    {"type": "section", "text": {"type": "mrkdwn", "text": sent_str}},
-                ],
+                blocks=[{"type": "section", "text": {"type": "mrkdwn", "text": sent_str}}],
```

Comment:
Please tell me that we were populating slack with "Danny Torrence left the following review for your property:"